test/format/wts.c

```diff
@@ -196,6 +196,40 @@ configure_timing_stress(char **p, size_t max)
     CONFIG_APPEND(*p, "]");
 }
 
+/*
+ * configure_debug_mode --
+ *     Configure debug settings.
+ */
+static void
+configure_debug_mode(char **p, size_t max)
+{
+    CONFIG_APPEND(*p, ",debug_mode=[");
+
+    if (GV(DEBUG_CHECKPOINT_RETENTION))
+        CONFIG_APPEND(*p, ",checkpoint_retention=%" PRIu32, GV(DEBUG_CHECKPOINT_RETENTION));
+    if (GV(DEBUG_CURSOR_COPY))
+        CONFIG_APPEND(*p, ",cursor_copy=true");
+    if (GV(DEBUG_CURSOR_REPOSITION))
```

Comment:
Let's get rid of `DEBUG_CURSOR_REPOSITION`. We intend it to remain disabled for now.

Reply:
yup, it has been addressed here: [2f48b80](https://github.com/wiredtiger/wiredtiger/pull/8608/commits/2f48b8050a474322acddd88eeca2ab638bc3032f)